pkg/autoops/api/progressive_rollout.go

```diff
@@ -850,9 +1010,13 @@ func (s *AutoOpsService) getFeature(
 	req *autoopsproto.CreateProgressiveRolloutRequest,
 	localizer locale.Localizer,
 ) (*featureproto.Feature, error) {
+	featureID := req.FeatureId
+	if req.Command != nil {
+		featureID = req.Command.FeatureId
+	}
```

Comment:
Just confirming.
Is there an issue that we need to check the featureID in the command, too?